vpr/src/place/place_delay_model.h

```diff
@@ -53,16 +52,14 @@ class DeltaDelayModel : public PlaceDelayModel {
     float delay(int from_x, int from_y, int /*from_pin*/, int to_x, int to_y, int /*to_pin*/) const override;
     void dump_echo(std::string filepath) const override;
 
-    void read(const std::string& /*file*/) override {
-        VPR_THROW(VPR_ERROR_ROUTE, "DeltaDelayModel::read unimplemented");
-    }
-    void write(const std::string& /*file*/) const override {
-        VPR_THROW(VPR_ERROR_ROUTE, "DeltaDelayModel::write unimplemented");
+    void read(const std::string& file) override;
+    void write(const std::string& file) const override;
+    const vtr::Matrix<float>& delays() const {
```

Comment:
Is it necessary to expose this implementation detail? Part of the motivation for making the PlaceDelayModel interface was to hide details like this.

Reply:
The reason was ease of implementation of OverrideDelayModel serialization, see https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/989#discussion_r330758264

I know how to remove this method if needed, but it is really about what to do about https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/989#discussion_r330758264 that changes what the "solution" is here.